sample/google-contacts.json

```diff
@@ -712,9 +712,9 @@
         "contactGroupMembership": { "contactGroupId": "myContacts" }
       }
     ],
-    "userDefined": [
+    "biographies": [
```

Comment:
https://developers.google.com/people/api/rest/v1/people#biography 

Are you sure about the array? Also it seems that they have a metaData field to keep ?

Reply:
about the array, yes: https://developers.google.com/people/api/rest/v1/people 

How can we keep metadata since our `note` field is only a string? (or I guess that we need to change the field rendering in the Contacts app)

Reply:
I created a card about this: https://trello.com/c/XG8aP1zz/1651-contacts-meilleure-synchro-du-champ-notes-biographies